src/elements/Step/Step.js

```diff
@@ -19,26 +19,31 @@ function Step(props) {
     className,
     'step',
   )
+  const getChildren = () => {
+    if (children) return children
+
+    return [<Step.Title>{title}</Step.Title>, <Step.Description>{description}</Step.Description>].map((item) => item)
+  }
+  const getContent = () => {
+    if (!icon) return <div className='content'>{getChildren()}</div>
+
+    return [iconPropRenderer(icon), <div className='content'>{getChildren()}</div>].map((item) => item)
+  }
```

Comment:
Hm, can you elaborate a bit on the goal here?  One thing I notice is that the `.map((item) => item)` looks to be redundant since it just returns every element in the array without modification:

![image](https://cloud.githubusercontent.com/assets/5067638/16993979/631a08d0-4e5a-11e6-82b5-d93e7e066ea0.png)